bioconda_utils/docker_utils.py

```diff
@@ -155,18 +158,6 @@ class DockerBuildError(Exception):
 
 
 
-def get_host_conda_bld():
-    """
-    Identifies the conda-bld directory on the host.
-
-    Assumes that conda-build is installed.
-    """
-    # v0.16.2: this used to have a side effect, calling conda build purge
-    # hopefully, it's not actually needed.
-    build_conf = utils.load_conda_build_config()
-    return build_conf.build_folder
```

Comment:
`.build_folder` is actually the `.../conda-bld/pkg_....` path if the `Config` would be initialized for some package `pkg` -- it just happens to be `.../conda-bld/` since the `pkg_...` part is empty for a `Config` not created for a certain package/recipe.
The new code has the correct `config.croot` (which is always only the `.../conda-bld` part.